test/test_spreadsheets.js

```diff
@@ -29,17 +29,17 @@ describe('scheduling', function() {
             options.extrema.iso_weekday = 1;
             options.extrema.reference_date = moment.utc("2016-04-07");
             var bounds = spreadsheets.get_round_extrema(options);
-            assert.equal(bounds.start.format(), "2016-04-04T11:00:00+00:00")
-            assert.equal(bounds.end.format(), "2016-04-11T11:00:00+00:00")
+            assert.equal(bounds.start.format(), "2016-04-04T11:00:00Z")
```

Comment:
I'm actually quite surprised by this change. These tests were passing on my system.  I wonder if the `.format` method is locale specific and we have different locales?  if so, we may want to consider passing in a specific format to each of these calls so that the locale doesn't change the test results.  

Again, don't worry about changing this pull request, Once this is merged I will pull and look into it.